### ISSUE TITLE
Animated NPC in new ensnare tower

### ISSUE THREAD

**Issue by Vendivar** [Vendivar/IslandTrollTribes2#158]

**Comment by Vendivar** [comment]
models/creeps/neutral_creeps/n_creep_forest_trolls/n_creep_forest_troll_high_priest.vmdl

**Comment by MNoya** [comment]
I made it cast the ensnare spell but its not currently tied to the attacks. Reopen this if we also want that

I used the _test model to show a ghost with the troll. Consider updating the main tower model to be on the same height as the first, and also remove that weird gnoll creep which is hanging below the tower (or do we want him to be the one to launch the attacks?)

**Comment by Vendivar** [comment]
Ideally he would be the one throwing the net. I moved the tower height
because it's suppose to be a smaller building.
On May 21, 2016 1:00 PM, "Martin Noya" <notifications@github.com> wrote:

I made it cast the ensnare spell but its not currently tied to the attacks.
Reopen this if we also want that

I used the _test model to show a ghost with the troll. Consider updating
the main tower model to be on the same height as the first, and also remove
that weird gnoll creep which is hanging below the tower (or do we want him
to be the one to launch the attacks?)

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/Vendivar/IslandTrollTribes2/issues/158#issuecomment-220797466>
